README.md

```diff
@@ -389,10 +389,11 @@ If you are running in pre-fork servers (such as Unicorn or Puma with multiple pr
 make sure you do this **before** the server forks. Otherwise, each child process may delete
 files created by other processes on *this* run, instead of deleting old files.
 
-**Define metrics before fork**: In order to share the same definition of metrics between
-your forks, you should make sure to define your metrics before to fork your process.
-Indeed, even if the content of the metric is stored in a file, the list of all metrics is
-stored in memory. Creating metrics after fork would lead to unexported files metrics.
+**Declare metrics before fork**: As well as deleting files before your process forks, you
+should make sure to declare your metrics then too. Because the metric registry is held in
```

Comment:
`declare your metrics then too`
I find that "then too" a bit unclear as to when it is.

Can we make it more redundant but clear, something like `you should also make sure to declare your metrics before the fork`